dev-util/pkgconf/pkgconf-1.5.3.recipe

```diff
@@ -6,28 +6,44 @@ COPYRIGHT="2011-2018 pkgconf authors"
 LICENSE="ISC"
 REVISION="1"
 SOURCE_URI="https://distfiles.dereferenced.org/pkgconf/pkgconf-$portVersion.tar.xz"
-CHECKSUM_SHA256="bab39371d4ab972be1d539a8b10b6cc21f8eafc97f617102e667e82bd32eb234"
-PATCHES="pkgconf-$portVersion.patchset"
+CHECKSUM_SHA256="d3468308553c94389dadfd10c4d1067269052b5364276a9d24a643c88485f715"
+ADDITIONAL_FILES="pkgconf-gcc2.sh"
 
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
 commandSuffix="$secondaryArchSuffix"
 if [ "$targetArchitecture" = x86_gcc2 ]; then
```

Comment:
I know, they're just for "consistancy" reasons :)

Reply:
Is it required to have `commandBinDir` outside? It's only used for x86_gcc2

Reply:
Oops, I just understood that you did not define `commandBinDir` in all cases because you only need it for `x86_gcc2`. So my comment is not really applicable...